gpgwrapper/backup.go

```diff
@@ -0,0 +1,20 @@
+package gpgwrapper
+
+import (
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/fluidkeys/fluidkeys/archiver"
+)
+
+func (g *GnuPG) BackupHomeDir(fluidkeysDir string, now time.Time) (string, error) {
+	cmd := "tar"
+	directory := archiver.DateStampedDirectory(fluidkeysDir, now)
```

Comment:
I *think* I'd move this up a level so that `BackupHomeDir` just makes the backup into the given directory (or even given filename)